### DISCUSSION TITLE
Link and Sync Device and Bulk Upload Albums

### DISCUSSION THREAD

**Discussion by BowChika** [immich-app/immich#3312] [Q&A]
Hi. I'm not sure if its a new feature request or this already exists so I'm just posting it as a question.

I have photos from before I started using Immich backed up in a central storage. Photos are in an album directory structure i.e. Photos in Camera folder represent the Camera Album. I already imported these photos to immich with --album parameter.

Now, I have 2 mobile phones that I use to take photos from. What I'm looking for is:
1- The be able to backup/upload camera photos from both my phones in to the same pre-existsing (CLI Bulk Upload) Camera album. Currently the CLI Bulk Upload Camera Album shows up separately on my mobiles phones. The phone's local Camera album is under "Albums on this Device". 
2- If 1 is achieved, it should automatically allow me to view Camera photos from both devices and the past on both my phones and on the web in a single Camera album.
3- If 1 is achieved, it will allow me to view camera photos from anywhere on the web as well. Currently the Web album do not show Camera photos from the On Device albums.

Is this possible somehow?

**Comment by bo0tzz** [comment]
Duplicate of #1678
